2022-02-09 15:56:00,122 INFO    MainThread:486034 [wandb_setup.py:_flush():71] setting env: {}
2022-02-09 15:56:00,122 INFO    MainThread:486034 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-09 15:56:00,122 INFO    MainThread:486034 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220209_155600-2td0f1lf/logs/debug.log
2022-02-09 15:56:00,122 INFO    MainThread:486034 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220209_155600-2td0f1lf/logs/debug-internal.log
2022-02-09 15:56:00,122 INFO    MainThread:486034 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7efedfc0f9d0>
2022-02-09 15:56:00,122 INFO    MainThread:486034 [wandb_init.py:init():404] calling init triggers
2022-02-09 15:56:00,122 INFO    MainThread:486034 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-09 15:56:00,123 INFO    MainThread:486034 [wandb_init.py:init():460] starting backend
2022-02-09 15:56:00,123 INFO    MainThread:486034 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-09 15:56:00,124 INFO    MainThread:486034 [backend.py:ensure_launched():216] starting backend process...
2022-02-09 15:56:00,135 INFO    MainThread:486034 [backend.py:ensure_launched():221] started backend process with pid: 486264
2022-02-09 15:56:00,137 INFO    MainThread:486034 [wandb_init.py:init():469] backend started and connected
2022-02-09 15:56:00,148 INFO    MainThread:486034 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-09 15:56:00,148 INFO    MainThread:486034 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-09 15:56:00,148 INFO    MainThread:486034 [wandb_init.py:init():533] updated telemetry
2022-02-09 15:56:00,190 INFO    MainThread:486034 [wandb_init.py:init():563] communicating current version
2022-02-09 15:56:00,852 INFO    MainThread:486034 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-09 15:56:00,854 INFO    MainThread:486034 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-09 15:56:01,070 INFO    MainThread:486034 [wandb_init.py:init():606] starting run threads in backend
2022-02-09 15:56:03,142 INFO    MainThread:486034 [wandb_run.py:_console_start():1810] atexit reg
2022-02-09 15:56:03,142 INFO    MainThread:486034 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-09 15:56:03,142 INFO    MainThread:486034 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-09 15:56:03,143 INFO    MainThread:486034 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-09 15:56:03,143 INFO    MainThread:486034 [wandb_init.py:init():633] run started, returning control to user process
2022-02-09 15:56:03,143 INFO    MainThread:486034 [wandb_run.py:_config_callback():956] config_cb None None {}
2022-02-09 15:56:03,159 INFO    MainThread:486034 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-09 15:56:03,160 INFO    MainThread:486034 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-09 15:56:21,584 INFO    MainThread:486034 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-09 15:56:21,585 INFO    MainThread:486034 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-09 15:56:21,586 INFO    MainThread:486034 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-09 15:56:24,826 INFO    MainThread:486034 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-09 15:56:24,927 INFO    MainThread:486034 [wandb_setup.py:_flush():71] setting env: {}
2022-02-09 15:56:24,927 INFO    MainThread:486034 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-09 15:56:24,927 INFO    MainThread:486034 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220209_155624-37j2hmc0/logs/debug.log
2022-02-09 15:56:24,928 INFO    MainThread:486034 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220209_155624-37j2hmc0/logs/debug-internal.log
2022-02-09 15:56:24,928 INFO    MainThread:486034 [wandb_init.py:init():404] calling init triggers
2022-02-09 15:56:24,928 INFO    MainThread:486034 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-09 15:56:24,928 INFO    MainThread:486034 [wandb_init.py:init():424] re-initializing run, found existing run on stack: 2td0f1lf
2022-02-09 15:56:24,929 INFO    MainThread:486034 [wandb_run.py:finish():1457] finishing run nelsonsw5/3DDet_Viz/2td0f1lf
2022-02-09 15:56:24,929 INFO    MainThread:486034 [jupyter.py:save_history():429] not saving jupyter history
2022-02-09 15:56:24,929 INFO    MainThread:486034 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-09 15:56:24,929 INFO    MainThread:486034 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-02-09 15:56:24,929 INFO    MainThread:486034 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-09 15:56:24,930 INFO    MainThread:486034 [wandb_run.py:_restore():1752] restore
2022-02-09 15:56:25,176 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-09 15:56:25,428 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-09 15:56:26,103 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11641
}

2022-02-09 15:56:26,207 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11641
}

2022-02-09 15:56:26,309 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11641
}

2022-02-09 15:56:26,411 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4798
  total_bytes: 11641
}

2022-02-09 15:56:26,515 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4798
  total_bytes: 11641
}

2022-02-09 15:56:26,617 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4798
  total_bytes: 11641
}

2022-02-09 15:56:26,719 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4798
  total_bytes: 11641
}

2022-02-09 15:56:26,822 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-09 15:56:26,926 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-09 15:56:27,028 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-09 15:56:27,130 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-09 15:56:27,401 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-09 15:56:27,855 INFO    MainThread:486034 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}
local_info {
}

2022-02-09 15:56:28,990 INFO    MainThread:486034 [wandb_run.py:_append_files():2180] logging synced files
